xds/internal/xdsclient/xdsresource/type_cds.go

```diff
@@ -56,6 +58,72 @@ type ClusterLBPolicyRingHash struct {
 	MaximumRingSize uint64
 }
 
+// OutlierDetection is the OutlierDetection configuration for a cluster.
+type OutlierDetection struct {
+	// Interval is the time interval between ejection analysis sweeps. This can
+	// result in both new ejections as well as addresses being returned to
+	// service. Defaults to 10000ms or 10s.
```

Comment:
Here and in other places: Does it really make sense to say both `10000ms` and `10s`. I think `10s` is much more readable than the former, and I would prefer just calling out: `Defaults to 10s`.

I understand both are mentioned in the gRFC, but I feel just one is good enough. Will leave it up to you though.